src/main/java/net/jextra/fauxjo/Table.java

```diff
@@ -143,14 +143,18 @@ public boolean delete( ConnectionSupplier cs, T bean )
     }
 
     public String buildBasicSelectStatement( String clause )
+            throws SQLException
```

Comment:
Adding `SQLException` to this method and `buildBasicSelect` above will cause existing code to break. Suggestion instead is to remove the throw spec, default the `fields` variable to `"*"`, and then wrap the `BeanDefCache` call in a try/catch block and swallow the exception. If an exception occurs, the method will continue and just use `"*"` as before. Likely if an exception happens here, it will manifest later in a place where the exception is _expected_ to be thrown, like actually executing the `select`.